scaffolder-templates/gitlab-merge-request/template.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: backstage.io/v1beta2
+kind: Template
+metadata:
+  name: gitlab-merge-request
+  title: GitLab Merge Request Action template
+  description: scaffolder v1beta2 template demo publishing to MR on existing git repository
+spec:
+  owner: backstage/techdocs-core
+  type: service
+
+  parameters:
+    - title: Fill in some steps
+      required:
+        - name
+      properties:
+        name:
+          title: Name
+          type: string
+          description: Unique name of the component
+          ui:autofocus: true
+          ui:options:
+            rows: 5
+        description:
+          title: Description
+          type: string
+          description: Description of the component
+        gitlab_project_id:
+          title: Gitlab Project ID
+          type: string
+          description: 'Provide your Gitlab Project ID #1234'
+        targetPath:
+          title: Target Path in repo
+          type: string
+          description: Name of the directory to create in the repository
+    - title: Choose a location
+      required:
+        - repoUrl
+      properties:
+        repoUrl:
+          title: Repository Location
+          type: string
+          ui:field: RepoUrlPicker
+          ui:options:
+            allowedHosts:
+              - gitlab.com
+  steps:
+    - id: fetch-base
+      name: Fetch Base
+      action: fetch:cookiecutter
```

Comment:
I think a template without cookiecutter would be better. What do you think?

Reply:
I agreed - all the other templates here are based on `fetch:template` which is what you get out of the box. `fetch:cookiecutter` comes with a few dependencies needed to install.

Reply:
have updated the changes as requested